12LessonTwelve/homework/index.html

```diff
@@ -1,10 +1,82 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Lesson 12 Homework</title>
+    <link href="https://fonts.googleapis.com/css?family=Lily+Script+One" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+
 </head>
+
 <body>
+    <section id="main">
+        <div class="top col-lg-12 col-md-12 col-sm-12">
+            <h3>Best Selling Products</h3>
+        </div>
+    </section>
+
+
+    <section id="bottom">
+        <div class="search col-lg-3 col-sm-6 col-xs-12">
+            <div class="box one">
+                <div class="green">$35.00</div>
+                <img src="img/lesson 12 one.jpg"></img>
+                <h4>Patient Ninja</h4>
+            </div>
+            <div class="dropbox">
+                <p>ADD TO CART</p>
+            </div>
+        </div>
+
+
+        <div class="search col-lg-3 col-sm-6 col-xs-12">
+            <div class="box two">
+                <div class="green">$35.00</div>
+                <img src="img/lesson 12 two.jpg"></img>
+                <h4>Happy Ninja</h4>
+            </div>
+            <div class="dropbox">
+                <p>ADD TO CART</p>
+            </div>
+        </div>
 
+
+        <div class="search col-lg-3 col-sm-6 col-xs-12">
+            <div class="box three">
+                <div class="sale">SALE!</div>
+                <div class="scratch">
+                    <div id="fifteen">$15.00</div>
+                    <div id="twelve">$12.00</div>
+                </div>
+                <img src="img/lesson 12 three.jpg" </img>
+                <h4>Ship Your Idea</h4>
+            </div>
+            <div class="dropbox">
+                <p>ADD TO CART</p>
+            </div>
+        </div>
+
+
+        <div class="search col-lg-3 col-sm-6 col-xs-12">
+            <div class="box four">
+                <div class="green">$18.00</div>
+                <img src="img/lesson 12 four.jpg" </img>
+                <h4>Flying Ninja</h4>
+            </div>
+            <div class="dropbox">
+                <p>ADD TO CART</p>
+            </div>
+        </div>
+    </section>
+
+
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
 </body>
+
 </html>
```

Comment:
Great job! The images are coming in broken for me, however. Make sure you have the image names correct. It looks like they might not be correct.